src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int Figure_Count = 5;
```

Comment:
Please follow the rules of constants naming:
```suggestion
    private static final int FIGURE_COUNT = 5;
```